libcudacxx/include/cuda/std/__internal/namespaces.h

```diff
@@ -104,6 +104,18 @@
 #  endif // not defined(_NVHPC_CUDA)
 #endif // not defined(CCCL_DISABLE_ARCH_DEPENDENT_NAMESPACE)
 
+// Host standard library namespaces
+#if _CCCL_HOST_STD_LIB(LIBSTDCXX)
```

Comment:
As discussed internally, we need to respect `_GLIBCXX_INLINE_VERSION ` to verify that there is an inline namespace